README.md

```diff
@@ -93,18 +105,10 @@ https://developer.salesforce.com/docs/component-library/documentation/lwc/get_st
 
 ### Testing
 
-You can run the included tests:
-
-`npm run:test:watch`
-
-## Dependencies
-
-This cookbook uses the [coveo.analytics.js](https://www.npmjs.com/package/coveo.analytics) library in order to track user interactions with the different components.
-
-It is included as a dependency in this project and served via a static resource within the different components of this repository.
-
-## Description of Files and Directories
+TBD
 
-## Add New Fields for Classification
+## How to Add New fields for Classification
 
-This cookbook suggests classifications for the standard case  field `reason`. To specify additional fields for classification, access the [caseAssistFlow folder](src/main/default/lwc/caseAssistFlow) and copy the sample components that are commented out in the `caseAssistFlow.html` and `caseAssistFlow.js` files as many times as necessary. If you are using the Case Review Form,  access the [caseReviewForm folder](src/main/default/lwc/caseReviewForm) and copy the sample components that are commented out in the `caseReviewForm.html` and `caseReviewForm.js` files as well. After modifying these files, access the Coveo Administration Console to update your Case Classification configuration accordingly. Make sure that all newly added fields are specified in your configuration. For more information, see https://docs.coveo.com/en/3328/service/manage-case-assist-configurations#configuring-the-case-classification-functionality. 
+This cookbook suggests classifications for the standard case fields Case Priority, Case Type, and Case Reason.
+To specify additional fields for classification, access the [Provide Details Screen](src/main/default/lwc/provideDetailsScreen) and copy the sample code that are commented out in the `provideDetailsScreen.html` and `provideDetailsScreen.js` files as many times as necessary. Remember to replace the placeholders `<SALESFORCE_API_FIELD_NAME>` and `<COVEO_FIELD_NAME>` with the Salesforce API name and the Coveo field name of the new field to predict, respectively.
```

Comment:
Add a sentence: 
You can find the Coveo Field Name in the [Fields ](https://docs.coveo.com/en/2036/index-content/about-fields) section of the Coveo Admin Console.